### PR TITLE
feat: Add getIsDataVisible function (STARGATE-1949)

### PR DESCRIPTION
**Pull request by IagoLast**

### CODE REVIEW

File: packages/pci-sdk-iframe/src/pages/app/app.service.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	try {
+		// If data is obtained just display it. We are done
+		const cardData = await apiClient.getCardData(cardId);
+		return cardData;
+	}
+	catch (err) {
+		// Otherwise we request a 2FA code
+		const { verificationId } = await apiClient.request2FACode();
+		// If request went well we init the verify2FA code and we'll fetch card data again with this code attached
+		return verify2FACode(cardId, verificationId, 'Enter the code we sent you (numbers only):');
```
Comment (lukewhyte) [inline packages/pci-sdk-iframe/src/pages/app/app.service.ts:21]: What happens when `apiClient.request2FACode` or `apiClient.verify2FACode` fails? Do we need to catch those errors?

File: packages/pci-sdk-iframe/src/types/IActionType.d.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+type IActionType = 'HIDE_DATA' | 'EMIT_VISIBILITY_MESSAGE' | 'SET_ERROR' | 'SET_CARD_DATA' | 'SET_LOADING' | 'SET_THEME';
```
Comment (lukewhyte) [inline packages/pci-sdk-iframe/src/types/IActionType.d.ts:1]: Is there any way (and value in) trying to use `IActionType` to set the type property in the `IAction.d.ts` interfaces?

File: packages/pci-sdk-iframe/src/pages/app/app.service.ts (hunk 1 above)
Comment (IagoLast) [reply packages/pci-sdk-iframe/src/pages/app/app.service.ts:21]: When this fails an error will be thrown and the whole function will fail and the .catch function will be executed by the hook.

In future versions we might want to throw different errors and react properly

File: packages/pci-sdk-iframe/src/types/IActionType.d.ts (hunk 2 above)
Comment (IagoLast) [reply packages/pci-sdk-iframe/src/types/IActionType.d.ts:1]: I've tried but no idea about how to do it :S

**Pull request merged**